indico/modules/rb_new/client/js/modules/roomList/filters/EquipmentForm.jsx

```diff
@@ -42,6 +42,7 @@ export default class EquipmentForm extends FilterFormComponent {
         this.state = {
             equipment: availableEquipment.filter(eq => selectedEquipment.includes(eq)),
             features: availableFeatures.map(f => f.name).filter(f => selectedFeatures.includes(f)),
+            activeIndex: -1,
```

Comment:
couldn't we just store a boolean whether the equipment details are visible or not?